src/components/AboutDashboard/components/AboutTheProtocolSection.tsx

```diff
@@ -27,6 +30,13 @@ export const AboutTheProtocolSection = () => {
             ),
           )}
         </div>
+
+        <ExternalLink href="https://github.com/jbx-protocol/juice-interface">
+          <AboutButton className="flex items-center gap-2">
```

Comment:
@wraeth-eth i think we should use antd button for the meantime, for styling consistency (slight differences in height, hover effect, dark/light mode etc.)

Reply:
Heads up: i switched over the existing aboutbuttons to antd buttons 🙏

Reply:
Thanks! Updated